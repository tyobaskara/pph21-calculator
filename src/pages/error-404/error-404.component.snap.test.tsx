import React from "react";
import renderer from 'react-test-renderer';
import Error404Page from './error-404.component';

jest
  .mock('../../components', () => ({
    Navigation: 'NavigationComponent'
  }));

test('Error404Page snaptest', () => {
  const component = renderer.create(
    <Error404Page/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});