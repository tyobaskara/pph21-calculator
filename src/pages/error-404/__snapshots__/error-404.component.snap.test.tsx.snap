// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404Page snaptest 1`] = `
<div>
  <NavigationComponent />
  <h1>
    Page Not Found
  </h1>
</div>
`;
